tests/integration/test_format_txts.py

```
@@ -0,0 +1,334 @@
+    test_file.write_text("Test content")
+
+    # Mock the subprocess execution to simulate an error
+    with patch("asyncio.create_subprocess_exec", new_callable=AsyncMock) as mock_exec:
+        # Set up the mock process to return an error
+        mock_process = AsyncMock()
+        mock_process.communicate.return_value = (b"", b"Error message")
+        mock_process.returncode = 1
+        mock_exec.return_value = mock_process
+
+        # Mock the CalledProcessError constructor to return a properly configured exception
+        original_cpe = subprocess.CalledProcessError
```

Ruff (0.8.2)

Local variable `original_cpe` is assigned to but never used

Remove assignment to unused variable `original_cpe`

(F841)